### ISSUE TITLE
File /SEUCK/gigaseuck.p00 missing

### ISSUE THREAD

**Issue by apurioe** [martinpiper/C64Public#1]
Hi Martin,

I was trying to follow http://tnd64.unikat.sk/SEUCK_Redux.html tutorial, and I got stuck when trying to compile, due to file /SEUCK/gigaseuck.p00 is missing in the repository. Is it the expected behavior?

Best regards,

Apurioe.

**Comment by martinpiper** [comment]
Yes. That game isn't released yet. Try one of the other vertically
scrolling data files in the directory.
